### DISCUSSION TITLE
Hangisi yalancı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#159] [Zeka Sorusu] [answered]
Mystogan , Filiz ve Forester den birisi hep yalan, birisi hep doğru birisi ise rastgele cevaplar veriyor. Bu kişilere evet-hayır cevaplı 3 soru sorma hakkınız var (istediğinize birden fazla istediğinize hiç sormayabilirsiniz). Sizin sorularınızı anlıyorlar ama cevap olarak sadece SONSUZ veya US diyorlar. Bunlardan birisi evet anlamına geliyor diğeri hayır anlamına ama siz henüz bunu da bilmiyorsunuz.
Hangisinin doğrucu, hangisinin yalancı hangisinin rastgeleci olduğunu anlamak için hangilerine ne sorarsınız?

**Comment by adasue** [comment, marked as answer]
Önce "5x5 25 mi yapar de diye sorsaydım SONSUZ mu derdin?" diye sorsaydım. Eğer SONSUZ'u doğru olarak alırsak doğrucu sonsuz derdi. yalancı da sonsuz derdi çünkü us deseydi doğruyu söylemiş olacaktı. Eğer sonsuz hayır anlamına gelseydi cümlenin akışına göre doğrucu gene sonsuz derdi ve yalancı da tekrar sonsuz derdi. Eğer cevabı olumsuz olan bir soru sorsaydım. Örneğin "5x5 20 yapar diye sorsaydım SONSUZ mu derdin?" dediğimde doğrucu ile yalancının ikisi de US derdi. Kelimelerin anlamlarının bir önemi yok. Filiz'e "Mystogan rastgeleci mi diye sorsam SONSUZ mu derdin? " diye sorduğumda eğer o rastgeleciyse cevabı çok önemli değil ama eğer yalancı ya da doğrucuysa ikisi de doğruysa SONSUZ yanlışsa US diyecek, diyelim ki onlardan biri ve US cevabını aldık. Sonsuz cevabını alırsak Forester'ın kesinlikle rastgeleci olma ihtimali kalmıyor. Us cevabını alırsak da Mystogan'ın kesinlikle rastgeleci olma ihtimali kalmıyor. Sonrasında rastgeleci olma ihtimali kalmayana "Sen yalancı mısın diye sorsam SONSUZ mu derdin?" diye sorduğumda yalancıysa sonsuz, doğrucuysa da us der. Verdiği cevaba göre doğrucu mu  yalancı mı anlarız ve tekrardan ona "Filiz rastgeleci mi diye sorsaydım SONSUZ mu derdin?" diye sorduğumuzda sonsuz veya us demesine göre herkesin kimliği ortaya çıkar.
